### PR TITLE
[docs] Minor readme edits

### PR DESCRIPTION
**Pull request by abiogenesis-now**
@kensimon @timothysc PTAL, just a quick change that @kstewart wanted.

(I need to rebase all of the history for my original fork, re-forking was just fastest for now).

### CODE REVIEW

**Comment by kensimon** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -7,13 +7,10 @@
 
 ## Overview
 
-Sonobuoy addresses the gap in tools for [Kubernetes][3] cluster introspection. It is a customizable, extendable, and cluster-agnostic way to record snapshots of your cluster's essential characteristics.
-
-Sonobuoy generates clear, informative reports about your cluster, regardless of your deployment details. Its selective data dumps of Kubernetes resource objects and cluster nodes allow for the following use cases:
+Sonobuoy addresses the gap in tools for [Kubernetes][3] cluster introspection. It is a customizable, extendable, and cluster-agnostic way to generate clear, informative reports about your cluster--regardless of your deployment details. Its selective data dumps of Kubernetes resource objects and cluster nodes allow for the following use cases:
```
Comment (timothysc) [inline README.md]: s/Its/It's a

Comment (abiogenesis-now) [inline README.md]: ^ @timothysc  do you mean that I should instead phrase this as "*It's a selective data dump* of Kubernetes resource objects and cluster nodes *, which allow* for the following use cases"?

Comment (abiogenesis-now) [inline README.md]: I used "its", as in Sonobuoy's multiple data dumps (but let me know if that's unclear)

**Comment by kensimon** [comment]
run tests

**Comment by kensimon** [comment]
run tests

**Comment by kensimon** [comment]
run tests

File: README.md (hunk 1 above)
Comment (timothysc) [inline README.md]: ahh.. /lgtm

**Pull request merged**